MAINTAINER_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+## Maintainer Guidelines
+
+### Releasing working code
+
+* run a full suite of actual integration tests before accepting a PR
```

Comment:
I think we should split this into `What to do when accepting a PR`, and `What to do when merging a release`.
E.g. on a PR we should validate:

```
1. Integration tests pass.
2. Code looks sound.
3. Travis/App Veyor Pass.
4. Rustfmt has been run.
```

And then for releases we can talk about anything that needs to be done there.

Reply:
For "merging a release" do you mean a PR like https://github.com/rusoto/rusoto/pull/883 ?

Reply:
Yea, that's where I was going 😄 sorry if that wasn't clear.